test/uk/gov/hmrc/gform/eval/EvaluationResultsSpec.scala

```diff
@@ -434,20 +446,34 @@ class EvaluationResultsSpec extends Spec with TableDrivenPropertyChecks {
       VariadicFormData.create(
         (toModelComponentId("1_addToListQuestion"), VariadicValue.One("0")),
         (toModelComponentId("2_addToListQuestion"), VariadicValue.One("1")),
+        (toModelComponentId("3_addToListQuestion"), VariadicValue.One("3")),
```

Comment:
Why this change? `addToListQuestion` can never have answer `3` it is yes/no question.

Reply:
some leftover from my experimentation, I will revert back